src/sentry/static/sentry/app/views/settings/incidentRules/create.tsx

```diff
@@ -29,9 +28,10 @@ type Props = {
  */
 class IncidentRulesCreate extends React.Component<Props> {
   handleSubmitSuccess = () => {
-    const {params, routes, router, location} = this.props;
+    const {router} = this.props;
+    const {orgId} = this.props.params;
 
-    router.push(recreateRoute('rules/', {params, routes, location, stepBack: -2}));
+    router.push(`/organizations/${orgId}/alerts/rules/`);
```

Comment:
You might need to add similar code to the `goBack()` function in the `ruleForm/index.tsx` file for the incident rule builder. Pressing 'Cancel' instead of 'Save Rule' will trigger that.